packages/react-query/src/shared/proxy/utilsProxy.ts

```diff
@@ -215,7 +215,7 @@ type DecorateProcedure<
 type DecorateRouter = {
   /**
    * Invalidate the full router
-   * @link https://trpc.io/docs/v10/useContext#query-invalidation
+   * @link https://trpc.io/docs/v10/useUtils#query-invalidation
```

Comment:
I know we dont have docs for v11 yet but this link will be broken?

v10/useUtils

Reply:
we shouldn't change v11 docs yet because we haven't generated a new version for v11, but good point